src/app/pages/auth/login/LoginFormResource.js

```diff
@@ -0,0 +1,17 @@
+import LoginFormView from './LoginFormView'
+import { withFinalForm } from '@ds-frontend/resource'
+import validate from './utils/validate'
+
+
+export default withFinalForm(
+  {
+    validate,
+  },
+  {
+    namespace: 'session',
+    endpoint: 'accounts/signin',
+  },
+  {
+    prefetch: false,
+  }
+)(LoginFormView)
```

Comment:
this file contains resource config and compose all together. please split it:
`LoginForm.js` - compose resource with view
`withLoginResource.js` - resource configuration